packages/@webex/plugin-meetings/src/config.ts

```diff
@@ -1,4 +1,5 @@
 import {
+  // @ts-ignore
```

Comment:
why do we need the ts-ignore here?

Reply:
The import is expecting a type file for `'@webex/common'`

TS Error:
`Module '"@webex/common"' has no exported member 'deviceType'. ts(2305)`

Reply:
Actually I looked into the code, here we are importing as `@webex/common` but we don't have `index.js` file at that location, instead it is at `@webex/common/src/`. I think due to webpack config it is able to pick `index.js` from src but for typescript it is not able to find the file at the expected location.

We can investigate this in a separate ticket. 
@arun3528 What is your thought on this?

Reply:
JIRA Ticket - https://jira-eng-gpk2.cisco.com/jira/browse/SPARK-393490